spec/fabricators/keyword_fabricator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Fabricator(:keyword) do
+  keyword FFaker::FreedomIpsum.word
```

Comment:
```suggestion
  keyword { FFaker::FreedomIpsum.word }
```
@aetheryna Should we add `{}` for the things that you want to make dynamic?
Because in the case of using `Fabricate.times(3, :keyword)`, it won't change that attribute for us 🥲

| Without `{}` | With `{}` |
| ------------ | --------- |
| ![ewa-payroll-web_–_job_position_fabricator_rb](https://user-images.githubusercontent.com/749672/196604738-ebf36779-6c0c-4971-9d41-0f37f6dbbd94.png) | ![ewa-payroll-web_–_job_position_fabricator_rb-2](https://user-images.githubusercontent.com/749672/196604767-b6663eeb-7cda-4287-8fa6-c6325961eb72.png) |

Reply:
Oh LOL, I didn't realize 😂 Updated here https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/f4ffa13498e6dba8963a0094691668859c6dd849